### ISSUE TITLE
Master Affiliate for MLM network

### ISSUE THREAD

**Issue by palepaul** [anujtenani/goaffpro#149]
A merchant has requested for an option, that will allow them to set a master affiliate, under which all the affiliates (who don't join using another affiliate's network link) will get assigned automatically.

**Comment by kennyWEE54** [comment]
Yes, I need this feature as well! Hope it will include the function to set product %, level % for master Affiliate.

**Comment by kennyWEE54** [comment]
Any update for these features? Thanks

**Comment by palepaul** [comment]
This option has been added (Multi Level > Default Sponsor) 

Guide: https://docs.goaffpro.com/multi-level-affiliate-network/enable-multi-level-marketing/set-default-sponsor

Thank you
